allensdk/brain_observatory/behavior/data_objects/metadata/subject_metadata/mouse_id.py

```diff
@@ -9,17 +9,19 @@
 class MouseId(DataObject, LimsReadableInterface, JsonReadableInterface,
               NwbReadableInterface):
     """the LabTracks ID"""
-    def __init__(self, mouse_id: int):
+    def __init__(self, mouse_id: str):
+        if not isinstance(mouse_id, str):
+            mouse_id = str(mouse_id)
         super().__init__(name="mouse_id", value=mouse_id)
 
     @classmethod
     def from_json(cls, dict_repr: dict) -> "MouseId":
         mouse_id = dict_repr['external_specimen_name']
-        mouse_id = int(mouse_id)
+        mouse_id = mouse_id
         return cls(mouse_id=mouse_id)
 
     @classmethod
-    def from_lims(cls, behavior_session_id: int,
+    def from_lims(cls, behavior_session_id: str,
```

Comment:
Sure about this? `BehaviorSessionId.value` is an int.

Reply:
Oops, sorry, trigger happy replace all. thanks for the catch.